releasenotes/notes/add-to-dict-from-dict-streamingchunk-f42dfbcca72e0465.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add `to_dict` and `from_dict` to StreamingChunk to make it consistent with our other dataclasses in having serialization and deserialization methods.
```

Comment:
Let's also update the release note to mention the other dataclasses that you added `to_dict` and `from_dict` to